doc/auto-discovery-docker.md

```diff
@@ -29,11 +29,11 @@ services:
     network_mode: "host"
     environment:
       SERVICE_HOST: 192.168.56.101
+      SERVICE_SERVERBINDADDR: 0.0.0.0
```

Comment:
This is only needed for the `camera notifications`, correct? If not using `camera notifications` then this isn't required.

Reply:
No actually I added the bind `0.0.0.0` because it is binding to `192.168.56.101`, and thus you can't access it via `127.0.0.1`, you HAVE to use the `192.168.56.101` address, so things like the `set-credentials.sh` script fail.